data/paper.org

```diff
@@ -92,8 +92,8 @@ the manner described above.
 The result is that:
 
 + The return curve's maximum decreases as $v_0$ increases. Thus, small-cap
-  investments have *high* potential returns (up to 500%), while large-cap
-  investments have *low* potential returns (as low as 20%).
+  investments have *high* potential returns (up to \sim310%), while large-cap
+  investments have *low* potential returns (as low as \sim5%).
```

Comment:
Actually, now that we have a true asymptotic function for max gains, we can say that large-cap gains, for the largest caps, max out at 20%, right? Thanks to the initial constant in the `max` function?

Reply:
I did say something to similar effect in the second example case